app/models/like.rb

```diff
@@ -0,0 +1,13 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :likes_counter
```

Comment:
- [ ]  You can use the after_save to call the #update_like_counter method every time a post has been liked